demo/server.js

```diff
@@ -1,14 +1,14 @@
-const express = require('express');
-const fs = require('fs');
-const initSqlJs = require('sql.js');
+const express = require("express");
```

Comment:
Use double quotes for text that people will read like messages and log statments. Use single quotes for text that is for machine processing.